Source/BottomView/StackView.swift

```diff
@@ -0,0 +1,138 @@
+import UIKit
+
+protocol ImageStackViewDelegate {
+  func imageStackViewDidPress()
+}
+
+class ImageStackView: UIView {
+
+  struct Dimensions {
+    static let imageSize: CGFloat = 70
+  }
+
+  var delegate: ImageStackViewDelegate?
```

Comment:
Please add `weak` here or we'll get a lot of pain afterwards :smiley: , like with `Pages`.